tophat/jenkins-stats-publisher-plugin
# feat: add jenkins run payload

added new class that will be our internal payload for information taken from a build
also added a git ignore for hpi files

DevQa
- add plugin to jenkins
- start a build
- at the end of build, go to system logs and check that payload was printed
